changes/ocd-3573.sql

```diff
@@ -0,0 +1,4 @@
+DROP TABLE IF EXISTS openchpl.certification_result_g1_macra;
+DROP TABLE IF EXISTS openchpl.certification_result_g2_macra;
+DROP TABLE IF EXISTS openchpl.pending_certification_result_g1_macra;
+DROP TABLE IF EXISTS openchpl.pending_certification_result_g2_macra;
```

Comment:
I'd appreciate a newline at the end of this file, as it helps with the "concatenation" of files during the PROD deployment, but it's not required